amazon-kinesis-client/src/test/java/software/amazon/kinesis/coordinator/DeterministicShuffleShardSyncLeaderDeciderTest.java

```diff
@@ -72,10 +73,18 @@ public void testLeaderElectionWithEmptyLeases() throws Exception {
         assertTrue("IsLeader should return true if no leases are returned", isLeader);
     }
 
+    @Test
+    public void testleaderElectionWithEmptyOwnerLeases() throws Exception {
+        List<Lease> leases = getLeases(5, true, true, true);
+        when(leaseRefresher.listLeases()).thenReturn(leases);
+        boolean isLeader = leaderDecider.isLeader(WORKER_ID);
+        assertTrue("IsLeader should return false if leases have no owner", isLeader);
```

Comment:
Test case says it is asserting false. but assert statement is validating it is true. Can you verify the test case here ?